content/participate/report-issue.adoc

```diff
@@ -31,9 +31,8 @@ functionality should be added.
 
 Other types (_Patch_, _Task_) are not used in JIRA.
 Neither will be explained here.
-If you want to propose a patch, create an issue using one of the 3 types above, and then create a link:https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request[pull request in GitHub].
-be explained here. If you want to propose a patch, create an issue of
-one of the 3 types above and then create a link:https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request[pull request in GitHub].
+If you want to propose a patch, create an issue using one of the 3 types above, and then create a pull request on GitHub link:https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request[as explained here].
+. Report issues with infrastructure hosted by the Jenkins project (jenkins.io, issues.jenkins.io, etc.) at link:https://github.com/jenkins-infra/helpdesk/issues/[the dedicated help desk].
```

Comment:
```suggestion
Report issues with infrastructure services hosted by the Jenkins project (jenkins.io, accounts.jenkins.io, etc.) at link:https://github.com/jenkins-infra/helpdesk/issues/[the dedicated help desk].
```

Removing the dot at the beginning, adding "services" so it makes more sense, replace issues.jenkins.io (which is not managed by the infrastructure team) by accounts.jenkins.io as more sensible example.

Reply:
Oh I guess I missed these few changes. Thanks for the correction! 
If it looks good, Should I close the PR?

Reply:
@whynesspower no need to close your PR, you can click on "commit suggestion", I'll review again your PR after that 🙂